Dockerfile

```diff
@@ -0,0 +1,40 @@
+FROM jupyter/base-notebook:latest
+
+LABEL maintainer="Fabian Brandt-Tumescheit <brandtfa@hu-berlin.de>"
+
+USER root
+RUN mkdir /var/lib/apt/lists/partial && \
+    apt-get update && \
+    apt-get install -y  --no-install-recommends cmake git build-essential && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+USER 1000
+RUN pip install --upgrade pip setuptools cython
+RUN pip install powerlaw
+RUN pip install sklearn
+RUN pip install seaborn
+RUN pip install pandas
+RUN pip install tabulate
+RUN pip install matplotlib
+RUN pip install networkx
+RUN pip install ipycytoscape
+
+USER root
+RUN mkdir -p ${HOME}/networkit
+COPY . ${HOME}/networkit/
+RUN mv ${HOME}/networkit/input ${HOME}
+RUN mv ${HOME}/networkit/notebooks ${HOME}
+RUN rm ${HOME}/notebooks/test_notebooks.py
+RUN chown -R 1000:100 ${HOME}/networkit
+RUN chown -R 1000:100 ${HOME}/input
+RUN chown -R 1000:100 ${HOME}/notebooks
```

Comment:
The gid looks off.

Reply:
The gid is correct (`jovyan:user` from Jupyter-base images). I exchanged for names instead of Ids to avoid confusion.